study-builder/tenants/cmi/change_password_email.html

```diff
@@ -0,0 +1,140 @@
+<html>
+<meta name="format-detection" content="telephone=no,date=no,address=no,email=no,url=no"/>
+<head>
+  <title>
```

Comment:
The `title` here is the subject line of the email. Copied it into here so we have some visibility to it.